src/cow-react/modules/wallet/api/hooks.ts

```diff
@@ -9,3 +9,7 @@ export function useWalletInfo(): WalletInfo {
 export function useWalletDetails(): WalletDetails {
   return useAtomValue(walletDetailsAtom)
 }
+
+export function useGnosisSafeInfo(): GnosisSafeInfo | undefined {
+  return useAtomValue(gnosisSafeInfoAtom)
+}
```

Comment:
the atom now is now moved to the generic hooks (they don't depend from any specific implementation, it just read from joitai state)